sled-agent/src/services.rs

```diff
@@ -3801,6 +3801,7 @@ impl ServiceManager {
             }
         };
 
+        info!(self.inner.log, "Setting up uplinkd service");
```

Comment:
I'm going to put https://github.com/oxidecomputer/omicron/pull/5593 for review soon and will be making substantial changes to this file. To avoid merge conflicts (I just finished fixing quite a few! ), do you mind removing these log messages from this PR and I promise to add them in mine?

Reply:
Done

Reply:
Thanks! :bowing_woman:

Reply:
Added here https://github.com/oxidecomputer/omicron/pull/5593/commits/2ea71b8fd613c676e9a17e4ac0453a5750da1092